website/events-listener/package.json

```diff
@@ -2,22 +2,26 @@
   "name": "@gear-js/event-listener",
   "version": "0.1.2",
   "description": "",
-  "main": "src/indes.js",
+  "main": "src/index.ts",
   "type": "module",
   "scripts": {
-    "start": "node dist/index.js",
-    "start:dev": "nodemon src/index.js",
-    "build": "cp -r src dist"
+    "start": "node --experimental-specifier-resolution=node dist/index.js",
+    "start:dev": "nodemon src/index.ts",
```

Comment:
You have to use ts-node with ts files. Nodemon will not work

Reply:
waiting its resolving
@xhjkl

Reply:
I'm having another experience:
```
$ yarn start:dev
[nodemon] 2.0.15
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: ts,json
[nodemon] starting `ts-node src/index.ts`
08:14:18 INFO [Main] App is running...

```

Reply:
Cool